src/v2/cookbook/dockerize-vuejs-app.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Dockerize Vue.js App
+type: cookbook
+order: 8
+---
+
+## Simple Example
+
+So you built your first Vue.js app using the amazing [Vue.js webpak template](https://github.com/vuejs-templates/webpack) and now you really want to show off with your colleagues by demonstrating that you can also run it in a Docker container.
+
+Let's start by creating a `Dockerfile` in the root folder of our project:
+
+```docker
+FROM node:9.8.0-alpine
+
+# install simple http server for serving static content
+RUN npm install -g http-server
+
+# make the 'app' folder the current working directory
+WORKDIR /app
+
+# copy both 'package.json' and 'package-lock.json' (if available)
+COPY package*.json .
```

Comment:
Did not try, but this should not work as well.
You cannot copy multiple files to `.`. It should be a folder and should end with `/`. Same problem should arise at line 32.